src/openrct2/Game.cpp

```diff
@@ -312,33 +331,39 @@ void game_update()
 {
     gInUpdateCode = true;
 
-    sint32 numUpdates;
+    uint32 numUpdates;
 
     // 0x006E3AEC // screen_game_process_mouse_input();
     screenshot_check();
     game_handle_keyboard_input();
 
     if (game_is_not_paused() && gPreviewingTitleSequenceInGame)
     {
-        title_sequence_player_update((ITitleSequencePlayer*)title_get_sequence_player());
+        title_sequence_player_update((ITitleSequencePlayer *) title_get_sequence_player());
     }
 
     // Determine how many times we need to update the game
-    if (gGameSpeed > 1) {
+    if (gGameSpeed > 1)
+    {
         numUpdates = 1 << (gGameSpeed - 1);
-    } else {
+    }
+    else
+    {
         numUpdates = gTicksSinceLastUpdate / GAME_UPDATE_TIME_MS;
-        numUpdates = clamp(1, numUpdates, GAME_MAX_UPDATES);
+        numUpdates = Math::Clamp(1u, numUpdates, (uint32) GAME_MAX_UPDATES);
```

Comment:
Maybe `GAME_MAX_UPDATES` can be made unsigned rather than having to cast it here?

Reply:
Tried it, but doesn't seem to work, sadly.